### DISCUSSION TITLE
Acik arttirma usulu bagis uygulamalarinin tek catida toplanmasi

### DISCUSSION THREAD

**Discussion by heyaygun** [acikkaynak/deprem-yardim-projesi#69] [Ideas]
Sporcular basta olmak uzere bircok gonullu acik arttirma ile satislar yapip Ahbap, Afad gibi kurumlara aktariyor. Acik arttirmalar mesaj yolu ile ilerliyor. Gerekli onaylar alinarak, belki Ahbap koordinatorlugunde bir noktada toplayabilir miyiz? 

Deprem sonrasi maddi yardimlar uzun bir sure icin gerekecek. Teyitli bir altyapimiz olursa dunya capinda yildizlara ulasilabilir. TeamDemiral ve Merih Demiral imzali futbolcu formalari ile iyi bir ornek oldular. Diger alanlarda da benzer isler organize edilebilir. Eminimki enstrumanini bagislayan, en ozel ramanini yazdigo kalemini, daktilosunu, bilgisayarini verecek kisiler olacaktir. Hatta diyorum ki bizim sektordeki arkadaslar da en iyi uygulamalarini yazdigi klavyelerini koyar...

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. [Bu](https://crypto.ahbap.org/tr) proje tarafımızca Ahbap için yapılmıştır.

**Reply by heyaygun** [reply]
> Merhaba arkadaşlar,
> Öncelikle fikriniz için teşekkürler. [Bu](https://crypto.ahbap.org/tr) proje tarafımızca Ahbap için yapılmıştır.

Teşekkürler @ramazansancar . Paylastigin link kripto bagis sayfasına yönleniyor.Bahsettigim fikir icin kısa bir dokuman hazırladım. Bakmak isterseniz linki bırakıyorum:   https://docs.google.com/document/d/1FcRQT_6wpJJTCGJfqIM-_ALL1Tb3cr3yPzVsgJh-1f8/edit#
